MerlinAU.sh

```diff
@@ -6400,7 +6470,7 @@ _PostRebootRunNow_()
    _DelPostRebootRunScriptHook_
 
    local theWaitDelaySecs=10
-   local maxWaitDelaySecs=600  #10 minutes#
+   local maxWaitDelaySecs=720 #12 minutes#
```

Comment:
Increase to 12 minutes due to the how long it took my last test run:

![image](https://github.com/user-attachments/assets/d8025d14-914f-4db3-828e-29b6fdeb6ec0)

Reply:
> Increase to 12 minutes due to the how long it took my last test run:
> 
Same questions as the other one with a similar increase.